src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
 
+class Header {
+  get logo() {
+    return f;
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the ' + `${f}`;
```

Comment:
PageObject should contain clickOnLogo method. The method should return a string 'Click on the' + logo from the Header class